js/main.js

```diff
@@ -0,0 +1,100 @@
+function Register() {
+    this.domElement = null;
+    this.entries = $(".entries");
+    this.inputBox = $(".newEntry");
+    this.totalEl = $(".total");
```

Comment:
You almost never want to use a whole-page jQuery selector `$(".whatever")` with components, because it will select elements from *other* instances of the same component.

Also, this won't work yet because when you instantiate the Register, it hasn't rendered yet, so the elements aren't on the page to select!